tests/jquery.flot-drawSeries.Test.js

```diff
@@ -290,5 +292,53 @@ describe('drawSeries', function() {
 
             expect(ctx.lineTo).toHaveBeenCalled();
         });
+
+        it('should draw bars for fillTowards infinity', function () {
+            series.datapoints.points = [150, 25, 50, 75, 200, 100];
+            series.bars.fillTowards = Infinity;
+
+            var xaxis = series.xaxis,
+                yaxis = series.yaxis,
+                expectedValue1 = xaxis.p2c(series.datapoints.points[0] - series.bars.barWidth / 2),
+                expectedValue2 = yaxis.p2c(maxy);
+
+            spyOn(ctx, 'lineTo').and.callThrough();
+
+            drawSeriesBars(series, ctx, plotOffset, plotWidth, plotHeight, null, getColorOrGradient);
+
+            expect(ctx.lineTo.calls.argsFor(0)).toEqual([expectedValue1, expectedValue2]);
```

Comment:
can you create a method that tests that all 3 (or 4 lineto's ) are the expected ones ?